pecan/decorators.py

```diff
@@ -32,7 +32,9 @@ def expose(template=None,
     access via HTTP, and to configure that access.
 
     :param template: The path to a template, relative to the base template
-                     directory.
+                     directory.  Can also be passed a string representing
+                     a special or custom renderer, such as ``'json'`` for
```

Comment:
so `'json'` is a custom (or special) renderer? I think we need a bit more prose (perhaps in a different place other than the docstring) because even though I know about it I am still not sure. For example if I create a custom renderer called `'custom'` can I use it in the same way?